docs/book/src/developer/architecture/controllers/cluster-resource-set.md

```diff
@@ -0,0 +1,14 @@
+# `ClusterResourceSet` Controller
+
+The `ClusterResourceSet` provide a mechanism for applying resources in a cluster once it is created.
```

Comment:
```suggestion
The `ClusterResourceSet` provide a mechanism for applying resources - e.g. pods, deployments, daemonsets - to a cluster once it is created.
```

Reply:
@killianmuldoon little confused here, resource types should be of Secrets/ConfigMaps right?

Reply:
got it. Thanks.